src/Rules/ForbiddenMultipleClassLikeInOneFileRule.php

```diff
@@ -17,7 +17,7 @@
 use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
 
 /**
- * @implements <FileNode>
+ * @implements Node<FileNode>
```

Comment:
Shouldn't this be `Rule<FileNode>`

Reply:
Yes. Running PHPStan on the code will report that.

Reply:
Fixed